docs/docs/self-service/flows/code/registration/index.js

```diff
@@ -0,0 +1,38 @@
+export const getFlowMethodPasswordWithErrors = {
```

Comment:
The files in docs were added on error?

Reply:
I do not know where this change come from 
I did not touch that file

Reply:
removed